example/app.config.ts

```diff
@@ -23,15 +23,15 @@ module.exports = () => ({
         backgroundColor: '#ffffff',
       },
       edgeToEdgeEnabled: true,
-      package: 'com.jakubswm.questlocation',
+      package: 'com.softwaremansion.horizon.demo',
```

Comment:
It should reflect the domain name.
```suggestion
      package: 'com.swmansion.horizon.demo',
```